deepchecks/vision/suites/default_suites.py

```diff
@@ -202,7 +199,7 @@ def model_evaluation(alternative_metrics: Dict[str, Metric] = None,
     --------
     >>> from deepchecks.vision.suites import model_evaluation
     >>> suite = model_evaluation()
-    >>> result = suite.run()
```

Comment:
Good idea adding max_samples to the example, can also be here and in data_integrity